client/src/components/common/applicant-info-nav.vue

```diff
@@ -1,43 +1,37 @@
 <template>
   <v-col cols="5" class="text-right py-0">
     <v-btn x-large
-           :id="`submit-back-btn-${isValid}`"
+           :id="`submit-back-btn-true`"
```

Comment:
I recommend you just remove "-true" here and elsewhere (and update styles/queries accordingly).
Otherwise this is a leftover artifact of a more complicated implementation. Simple is better! 😁